src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -893,7 +893,9 @@ javadocTag: AUTHOR_LITERAL (WS | NEWLINE)* description?
 
       | VERSION_LITERAL (WS | NEWLINE)* description?
 
-      | CUSTOM_NAME (WS | NEWLINE)* description?
+      | CUSTOM_NAME (WS | NEWLINE)*
+        ({!isNextJavadocTag()}? LEADING_ASTERISK | htmlComment
+        | CDATA | NEWLINE | text | javadocInlineTag | htmlElement)*
```

Comment:
I am not sure whether the block tag should be affected by this issue.
@romani , please share your opinion.